css/style.css

```diff
@@ -652,44 +845,76 @@ button {
   gap: 16px;
 }
 
-/**************************** Partners section *****************************/
+/*********************** Partners section ***********************/
 .partners {
   padding-top: 50px;
   padding-bottom: 50px;
   background-color: var(--main-background-color);
+  position: relative;
+  overflow: hidden;
+}
+
+.partners::before {
+  content: "";
+  width: 284px;
+  height: 284px;
+  background-image: url("../img/bg-s/star-small.svg");
+  background-size: 284px;
+  background-repeat: no-repeat;
+  transform: rotate(165deg);
+  position: absolute;
+  bottom: 30px;
+  left: -160px;
+  opacity: 80%;
 }
 
 .partners-cards {
   margin-top: 58px;
-  text-align: center;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 30px;
+  justify-content: center;
 }
 
-.partner-banner {
+.partners-card-item {
+  width: 210px;
+  height: 100px;
   padding: 20px 24px;
-  margin-bottom: 30px;
+}
+
+.partner-banner {
+  display: block;
 }
 
 .banner-bg {
   background-color: var(--secondary-color);
 }
 
-/************************** FAQs section *********************************/
+/************************** FAQs section ***********************/
 
-.faq-box {
-  padding-top: 50px;
-  padding-bottom: 50px;
+.faq-box-section {
   background-color: var(--main-background-color);
+  margin-top: -200px;
 }
 
 .faq-title {
   margin-bottom: 60px;
 }
 
+.faq-box {
+  position: relative;
+  top: 250px;
```

Comment:
Because I didn't find a better solution how to put  .faq-box between two backgrounds and prevent covering  next section (.cta-homepage section) when details expand by pressing +. 
I have moved negative margin to .faq-box. 
But I'll be happy to learn a better way ))

Reply:
Maybe I'm missing something. Usually, if we have one background running over another, then we really use a negative margin. That is, we don't need the top value here. Then the cta-homepage will not need such a large padding, since everything will follow each other. And after that you need to add only necessary paddings or margins between the sections

<img width="1295" alt="Снимок экрана 2024-09-02 в 2 52 29 PM" src="https://github.com/user-attachments/assets/b028517e-6166-4c64-b355-8ff2711122b8">
<img width="1390" alt="image" src="https://github.com/user-attachments/assets/fc731f0e-baab-4287-b77c-871eaeb6c462">